UKHomeOffice/cerberus-service
# COP-4592 Add issue target to task list page & refactor

## Description
- add issued target tab
- add tests
- refactor to make scenarios clearer

## To Test

- go to task list page
> you should see tasks with 'claim' buttons
- click `claim`
> you should be taken to that tasks task details page (note the businessKey)
- return to task list
- click on `in progress` tab
> you should see the task you just claimed with an `unclaim` link
> you should see other tasks with no unclaim link (they are not assigned to you)
- go to your task
- click on `issue target` green button
- complete the form
- go back to task list
- go to `issued target` tab
> you should see your task with no unclaim or claim button
- go to `complete` tab
> you should see some completed tasks with no unclaim or claim button (we can only complete these via cockpit so these were completed earlier)

## Developer Checklist

\* Required
